test/unit/connection/ClientMessageReaderTest.js

```diff
@@ -99,3 +100,154 @@ describe('ClientMessageReaderTest', function () {
         expect(reader.read()).to.be.equal(null);
     });
 });
+
+describe('PortedClientMessageReaderTestFromJava', function () {
```

Comment:
The suite name should be the same as well. You should change the one that was existed before.

Reply:
Having same name for 2 describe methods looks weird. Thats why I changed it. Done.

Reply:
Can you change the one above?